stagingUtils.js

```diff
@@ -247,11 +240,9 @@ module.exports = {
         });
     });
   },
-  async getGitPatchFromCommits(firstCommit, lastCommit) {
-    // need to delete patch file?
+  async getGitPatchFromCommits() {
     return new Promise((resolve, reject) => {
-      if (lastCommit === null) {
-        const patchCommand = 'git show HEAD > myPatch.patch';
+        const patchCommand = `git diff master...HEAD --ignore-submodules > myPatch.patch`;
```

Comment:
It would be better to pipe this command's output directly and not use a temp file, especially a temp file with a fixed name, but this is the existing behavior so I guess it's okay

Reply:
You could get around fixed name by throwing a {{$$}} in there like: {{> myPatch.$$.patch}}. It's not perfect, but it avoids clobber potential.

Reply:
The existence of this file is important for the build flow so please do not remove it! Once we move away from makefiles we can revisit. Thanks.